.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -8,28 +8,36 @@ assignees: ''
 
 <!--- NOTE: PLEASE FILL OUT TEMPLATE RATHER THAN DELETING --->
 
-**Describe the bug**
+## Describe the bug
```

Comment:
I'm not sure about this change. It converts the text to a header instead of bold which imho is better. Its a nit but the change seems unrelated unless im missing something.

I guess this was a lint? Maybe we can disable it?

Reply:
Yes it was a lint: https://github.com/markdownlint/markdownlint/blob/main/docs/RULES.md#md036---emphasis-used-instead-of-a-header

Why do think bold is better? Do you prefer how it's displayed? Semantically I think the lint is right here. We could use a lower heading level though, so that it looks similar to bold text.

Reply:
Changed it to h4 heading. Please let me know what you think.